Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+  agent {
+    docker {
+      image 'golang:1.9.2'
+    }
+  }
+
+  stages {
+    stage('Test') {
+      steps {
+        sh """
+        mkdir -p /go/src/github.com/influxdata
+        cp -a $WORKSPACE /go/src/github.com/influxdata/influxql
+
+        cd /go/src/github.com/influxdata/influxql
+        go get -v -t
+        go test -v | tee $WORKSPACE/test-results.log
+        """
+      }
+
+      post {
+        always {
+          sh """
+          if [ -e test-results.log ]; then
+            go get github.com/jstemmer/go-junit-report
+            go-junit-report < $WORKSPACE/test-results.log > test-results.xml
+          fi
+          """
+          junit "test-results.xml"
```

Comment:
Normally, if `test-results.log` didn't exist and we don't create `test-results.xml`, then the `junit` call will fail. But if `test-results.xml` exists in the workspace from a previous run, then `junit` may operate on stale results. I think the `steps` block above needs to delete `test-results.log` and `test-results.xml` if they exist when we start the job.

Reply:
Done.